core/src/main/java/nu/yona/server/subscriptions/service/BuddyConnectResponseMessageDto.java

```diff
@@ -136,10 +136,7 @@ MessageActionDto handleAction_Process(UserDto actingUser, BuddyConnectResponseMe
 			}
 			else
 			{
-				buddyService.setBuddyAcceptedWithSecretUserInfo(actingUser.getOwnPrivateData().getUserAnonymizedId(),
-						connectResponseMessageEntity.getBuddyId(),
-						connectResponseMessageEntity.getRelatedUserAnonymizedId().get(),
-						connectResponseMessageEntity.getSenderNickname());
+				buddyService.setBuddyAcceptedWithSecretUserInfo(actingUser, connectResponseMessageEntity);
```

Comment:
Isn't it a bit weird to pass an entity to a service? I know that we do that more often, but I would rather pass a DTO.